flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -492,6 +492,21 @@ private static int getDrawableResourceId(Context context, String name) {
         return context.getResources().getIdentifier(name, DRAWABLE, context.getPackageName());
     }
 
+    private static byte[] objectDataToByteArray(Object data) {
```

Comment:
Can you apply the `@SuppressWarnings("unchecked")` annotation here? Not having the annotation here would result in warnings for all consumers of this library around unsafe operations. This should also allow for removing this annotation from the `getBitmapFromSource` method.

Regarding the name of the method, could this be something like `castObjectToByteArray`? Reason I'm proposing this is to improve readability since methods are actions and the name is lacking a verb to describe the action

Reply:
No problem, I applied all of your suggestions - Java and it's convention are not my strong suit.